internal/devices/drivers.go

```diff
@@ -70,9 +65,14 @@ func InstallDrivers(ctx context.Context, vm *uvm.UtilityVM, share string, gpuDri
 	if err != nil {
 		return closer, fmt.Errorf("failed to add SCSI disk to utility VM for path %+v: %s", share, err)
 	}
+	closer = mount
+	uvmPathForShare = mount.UVMPath
 
 	// construct path that the drivers will be remounted as read/write in the UVM
-	driverGUID, err := guid.NewV4()
+
+	// 914aadc8-f700-4365-8016-ddad0a9d406d. Random GUID chosen for namespace.
+	ns := guid.GUID{Data1: 0x914aadc8, Data2: 0xf700, Data3: 0x4365, Data4: [8]byte{0x80, 0x16, 0xdd, 0xad, 0x0a, 0x9d, 0x40, 0x6d}}
```

Comment:
Can we make this a var at the top of the file?

Reply:
If we're only using it in one place, I think that actually reduces readability. The reader has to validate it's not being used in other places to understand it's scope, then.

Reply:
That seems reasonable to me.